web-animations-2/Overview.bs

```diff
@@ -969,6 +972,30 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  The [=animation/progress=] of an [=animation=] is calculated as follows:
+
+  <dl class=switch>
+
+  : If <em>any</em> of the following are true:
+    * the animation does not have an [=animation/associated effect=], or
+    * the animation's associated effect's [=end time=] is infinite, or
```

Comment:
What's the thinking behind making infinite → null as opposed to zero? I think zero might be more useful overall since it represents where the animation is actually up to.